common/models/user.js

```diff
@@ -709,7 +698,7 @@ module.exports = function(User) {
       });
       if (!idsToExpire.length) return next();
       AccessToken.deleteAll({ userId: { inq: idsToExpire }}, function(err, info) {
-        if (err) return next(err);
+        if (err) return next();
```

Comment:
This is dropping (ignoring) any `deleteAll` errors. Is that intentional?

I would personally use the following implementation:

```js
AccessToken.deleteAll({ userId: { inq: idsToExpire }}, next);
```